components/butterfly/src/error.rs

```diff
@@ -43,29 +45,31 @@ pub enum Error {
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         let msg = match *self {
-            Error::BadMessage(ref err) => format!("Bad Message: {:?}", err),
-            Error::CannotBind(ref err) => format!("Cannot bind to port: {:?}", err),
+            Error::BadMessage(ref err) => format!("Bad Message, {:?}", err),
```

Comment:
I use colon for that *everywhere* in this code base, and I think `:` is more appropriate than a comma, since you're not completing a sentence. I'm a 2/10 about how much it matters, but we should decide, and my preference is colon's not comma's.